app/models/sources/static.rb

```diff
@@ -0,0 +1,50 @@
+module Sources
+  class Static < Source
+    def decommission_proxy(proxy)
+      # Do nothing. The Proxy instance is already going to be deleted.
+    end
+    
+    def provision_proxies(desired_proxy_count, site)
+      # Do nothing. There's no automatic way to provision proxies from a static list.
+    end
+    
+    def validate_config!
+      true  # No config to validate
+    end
+    
+    def max_proxies
+      0  # Never try to provision extra proxies from a static list
+    end
+    
+    def load_from_file(path, site = Site::NoSite)
```

Comment:
At the moment, only from the console. Within the next two weeks or so, I'll be implementing an actual user interface for adding and removing proxies from static sources. That work will use robust, tested code paths, making this method unnecessary (and allowing me to remove it).